### PR TITLE
A few mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/advancement/Advancement.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS q net/minecraft/advancement/SimpleAdvancement
+CLASS q net/minecraft/advancement/Advancement
```
Comment (liach) [inline mappings/net/minecraft/advancement/Advancement.mapping:1]: Uh, when did we decide this

File: mappings/net/minecraft/entity/ai/goal/MoveToNearbyPointOfInterestGoal.mapping [hunk 2]
```diff
@@ -0,0 +1 @@
+CLASS and net/minecraft/entity/ai/goal/MoveToNearbyPointOfInterestGoal
```
Comment (liach) [inline mappings/net/minecraft/entity/ai/goal/MoveToNearbyPointOfInterestGoal.mapping]: Should be wandering around a point of interest.

File: mappings/net/minecraft/util/math/ChunkSectionPos.mapping [hunk 3]
```diff
@@ -29,6 +37,9 @@ CLASS fo net/minecraft/util/math/ChunkSectionPos
 		ARG 1 y
 	METHOD a from (Lev;)Lfo;
 		ARG 0 pos
+	METHOD a stream (Lfo;I)Ljava/util/stream/Stream;
+		ARG 0 pos
```
Comment (liach) [inline mappings/net/minecraft/util/math/ChunkSectionPos.mapping]: center

File: mappings/net/minecraft/world/chunk/ChunkPos.mapping [hunk 4]
```diff
@@ -22,10 +22,12 @@ CLASS bgq net/minecraft/world/chunk/ChunkPos
 		ARG 3 chunkRelativeZ
 	METHOD a getPackedX (J)I
 		ARG 0 pos
-	METHOD a streamPositions (Lbgq;I)Ljava/util/stream/Stream;
+	METHOD a stream (Lbgq;I)Ljava/util/stream/Stream;
 		ARG 0 pos
```
Comment (liach) [inline mappings/net/minecraft/world/chunk/ChunkPos.mapping]: center

File: mappings/net/minecraft/advancement/Advancement.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/advancement/Advancement.mapping:1]: There are no other advancement classes or interfaces. This class is named wrongly, every advancement is a `SimpleAdvancement`

File: mappings/net/minecraft/entity/ai/goal/MoveToNearbyPointOfInterestGoal.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/entity/ai/goal/MoveToNearbyPointOfInterestGoal.mapping]: As far as I see, this should be a replacement for the old iron golem wander goal for fixing iron golem leaving village bug, so we should say `WanderAroundPointOfInterestGoal`

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
Mind cherry picking https://github.com/FabricMC/yarn/pull/629/files#diff-07537dd8de0831eabd1ddda8a1dafce9R6 and https://github.com/FabricMC/yarn/pull/629/files#diff-c5b04d3967fe4dfcc81500d81fcf757cR65 ?

**Comment by Runemoro** [comment]
@liach Done. Sorry, I didn't realize you had done a PR on the prerelease classes too.

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/entity/ai/brain/task/VillagerTaskListProvider.mapping [hunk 5]
```diff
@@ -1,9 +1,9 @@
 CLASS alm net/minecraft/entity/ai/brain/task/VillagerTaskListProvider
-	METHOD a makeFreeFollowTasks ()Lcom/mojang/datafixers/util/Pair;
+	METHOD a getFreeFollowTask ()Lcom/mojang/datafixers/util/Pair;
 	METHOD a getPlayTasks (F)Lcom/google/common/collect/ImmutableList;
 	METHOD a getCoreTasks (Lavk;F)Lcom/google/common/collect/ImmutableList;
 		ARG 0 profession
-	METHOD b makeBusyFollowTasks ()Lcom/mojang/datafixers/util/Pair;
+	METHOD b getBusyFollowTask ()Lcom/mojang/datafixers/util/Pair;
```
Comment (liach) [inline mappings/net/minecraft/entity/ai/brain/task/VillagerTaskListProvider.mapping]: I suggest using `create` instead as these methods are making new instances of tasks every time they are called.

Comment (Runemoro) [reply mappings/net/minecraft/entity/ai/brain/task/VillagerTaskListProvider.mapping]: Yes, you're right, `create` or `make` would be better for all methods.

Comment (Runemoro) [reply mappings/net/minecraft/entity/ai/brain/task/VillagerTaskListProvider.mapping]: Done.

File: mappings/net/minecraft/entity/ai/brain/task/WanderIndoorsTask.mapping [hunk 6]
```diff
@@ -0,0 +1,2 @@
+CLASS ajy net/minecraft/entity/ai/brain/task/WanderIndoorsTask
+	FIELD a speed F
```
Comment (NeunEinser) [inline mappings/net/minecraft/entity/ai/brain/task/WanderIndoorsTask.mapping:2]: WIth that, it'd be completely deobfuscated, why not adding it right away?
```
	METHOD <init> (F)V
		ARG 1 speed
```

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**